src/cpp/ext/otel/otel_plugin.h

```diff
@@ -66,6 +66,15 @@ class LabelsIterable {
 // through the OpenTelemetry plugin.
 class LabelsInjector {
  public:
+  // Some optional labels may only be set on client or server. We use this enum
+  // to specify the entity of the caller for the APIs that need this
+  // information.
+  enum class Entity {
```

Comment:
Throughout gRPC Core, we've just been using a simple `is_client` bool for this. If we were to do this, I would prefer doing it all over.

Reply:
+1 -- let's just use a bool here.  We're not expecting to have any context other than client or server in the future, so there's no need to leave room for future options.

Reply:
Done.